pennylane/ops/functions/simplify.py

```diff
@@ -62,10 +67,59 @@ def simplify(op: Operator):
     >>> qml.simplify(op)
     Adjoint(RX)(1.5707963267948966, wires=[0]) + Adjoint(PauliX)(wires=[0])
 
+    Moreover, ``qml.simplify`` can be used to simplify QNodes or quantum functions:
+
+    >>> dev = qml.device("default.qubit", wires=2)
+    >>> @qml.simplify
+        @qml.qnode(dev)
+        def circuit():
+            qml.adjoint(qml.prod(qml.RX(1, 0) ** 1, qml.RY(1, 0), qml.RZ(1, 0)))
+            return qml.probs(wires=0)
+    >>> circuit()
+    tensor([[0.64596329, 0.35403671]], requires_grad=True)
+    >>> list(circuit.tape)
```

Comment:
I think this shows we need to think about how to to draw circuits with symbolic operators, but that is a separate problem.